<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
  font-family: &quot;Courier New&quot;;
  background-color: #FFFFFF;
}
QGroupBox {
  font-family: &quot;Courier New&quot;;
  font-size: 13px;
  background-color: rgba(248, 249, 250, 1);
}
QLineEdit {
  padding: 4px 0;
  border: 1px solid #ced4da;
  border-radius: 8px;
  margin-left: 8px;
  padding-left: 4px;
  background-color: #FFFFFF;
  color:#212529;
}
QPlainTextEdit {
	background-color: #FFFFFF;
  color:#212529;
}
QLabel {
  font-family: &quot;Courier New&quot;;
  font-size: 13px;
  color: rgb(90, 93, 97);
}
QRadioButton {
  font-family: &quot;Courier New&quot;;
  font-size: 13px;
  color: #212529;
}
QComboBox {
  font-family: &quot;Courier New&quot;;
  font-size: 13px;
  color: #212529;
}
QPushButton {
  color: #212529;
}

QCheckBox{
  color: #212529;
}
QPushButton#runbot_btn {
	margin-top: 8px;
	border-radius: 8px;
	min-height: 24px;
	background-color: #FFA500;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QWidget" name="container_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>12</number>
         </property>
         <item>
          <widget class="QWidget" name="data_select_container" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>8</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="data_select_layout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QComboBox" name="data_combobox">
                <item>
                 <property name="text">
                  <string>Select table</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="data_open_btn">
                <property name="text">
                 <string>Open</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="thread_num_container" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>8</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QSpinBox" name="thread_num_input"/>
              </item>
              <item>
               <widget class="QLabel" name="thread_num_label">
                <property name="font">
                 <font>
                  <family>Courier New</family>
                  <pointsize>-1</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">margin: 0;</string>
                </property>
                <property name="text">
                 <string>Thread nums</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="threads_container_layout"/>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="group_key_container" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="group_key_label">
                <property name="font">
                 <font>
                  <family>Courier New</family>
                  <pointsize>-1</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">margin: 0;</string>
                </property>
                <property name="text">
                 <string>Group keywords</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="group_key_input">
                <property name="styleSheet">
                 <string notr="true">margin: 0;
padding-left: 4px;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="ignore_group_key_container" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>8</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QLabel" name="ignore_group_key_label">
                <property name="font">
                 <font>
                  <family>Courier New</family>
                  <pointsize>-1</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">margin: 0;</string>
                </property>
                <property name="text">
                 <string>Ignore group keywords</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="ignore_group_key_input">
                <property name="styleSheet">
                 <string notr="true">margin: 0;
padding-left: 4px;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="runbot_btn">
           <property name="text">
            <string>Run bot</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
